angr/analyses/reaching_definitions/reaching_definitions.py

```diff
@@ -38,7 +38,7 @@ class ReachingDefinitionAnalysis(ForwardAnalysis, Analysis):  # pylint:disable=a
 
     def __init__(self, subject=None, func_graph=None, max_iterations=3, track_tmps=False,
                  observation_points=None, init_state=None, init_func=False, cc=None, function_handler=None,
-                 current_local_call_depth=1, maximum_local_call_depth=5, observe_all=False):
+                 current_local_call_depth=1, maximum_local_call_depth=5, observe_all=False, visited_blocks=[]):
```

Comment:
Do not use a container as the default value of a function parameter since the default value will get changed during runtime (yes this is Python's fault). Use `None` instead.